src/backend/main.ts

```diff
@@ -1963,26 +2007,4 @@ ipcMain.on('killOverlay', () => {
   closeOverlay()
 })
 
-ipcMain.handle('completeHyperPlayQuest', async () => {
-  const authSession = session.fromPartition('persist:auth')
-
-  const cookies = await authSession.cookies.get({
-    url: DEV_PORTAL_URL
-  })
-
-  const cookieString = cookies
-    .map((cookie) => `${cookie.name}=${cookie.value}`)
-    .join('; ')
-
-  const response = await fetch(`${DEV_PORTAL_URL}/api/hyperplay-quest`, {
-    method: 'POST',
-    headers: {
-      Cookie: cookieString
-    }
-  })
-
-  if (!response.ok) {
-    const error = await response.json()
-    throw new Error(`Failed to complete summon task: ${error}`)
-  }
-})
+ipcMain.handle('completeHyperPlayQuest', completeHyperPlayQuest)
```

Comment:
are these needed?

this ipc Main handler, 
the type `completeHyperPlayQuest: () => Promise<void>` and 
```
export const completeHyperPlayQuest = async () =>
  ipcRenderer.invoke('completeHyperPlayQuest')
```

Reply:
no they're not, I'll remove